tests/xtimer_msg/tests/01-run.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+
+def testfunc(child):
+    # 1st check for periodic 2s Hello World message, i.e., 2 output + 1 msg
+    for _ in range(7):
+        child.expect(r"sec=\d+ min=\d+ hour=\d+")
+        child.expect(r"sec=\d+ min=\d+ hour=\d+")
+        child.expect(r"now=\d+:\d+ -> every 2.0s: Hello World")
```

Comment:
Just thinking of it: what about adding some high level verification of the timing. Using python `time` module I think it should be possible to check that the messages are effectively displayed every 2s (and 5s below).

Reply:
Our (reviewing guidelines)[https://github.com/RIOT-OS/RIOT/wiki/Guidelines-for-Reviewing-a-Pull-Request) state clearly that "while you're at it" is bad practice. Get this merged, improve later!

Also, there was consensus that we check (x)timer's accuracy in *one* test designed for that. All other tests then trust the node's notion of time. Otherwise we (unneccessarily) complicate the tests.

Reply:
@kaspar030, I don't think "while you're at it" applies here: the script is added here, so we why not make it "right" from the start. On the other hand I also agree with you, that we don't want to verify xtimer accuracy here but rather that messages arrive in a certain, expected pattern. Which should be sufficiently verified by looking for 2 + 1 "Hello world" message in the first case and 5 + 1 "This is a Test" messages in the second case.

TL;DR: lets leave as is for now.

Reply:
This was not a blocking comment from my side but rather a question. Thanks for the answers :)